src/gui/plugins/plotting/Plotting.cc

```diff
@@ -382,6 +448,20 @@ void Plotting ::Update(const ignition::gazebo::UpdateInfo &_info,
       if (comp)
         this->SetData(component.first, comp->Data());
     }
+    else if (typeId == components::Light::typeId)
+    {
+      auto comp = _ecm.Component<components::Light>(entity);
+      if (comp){
+        msgs::Light lightMsgs = convert<msgs::Light>(comp->Data());
+        this->SetData(component.first, lightMsgs);
+      }
+    }
+    else if (typeId == components::LightCmd::typeId)
+    {
+      auto comp = _ecm.Component<components::LightCmd>(entity);
+      if (comp)
+        this->SetData(component.first, comp->Data());
+    }
```

Comment:
Do we need to have `LightCmd` components in Plotting and ComponentInspector? From my understanding they would be created on change request then deleted immediately afterwards?

Reply:
[reopening since the same component was also added in `ComponentInspector`, just to confirm whether it is needed or not]

Reply:
I think we don't need this in both places, we cannot plot this and this is not going to be shown in the Component inspector.